src/Rings/MPolyQuo.jl

```diff
@@ -331,7 +331,7 @@ julia> R, (x, y, z) = polynomial_ring(QQ, ["x", "y", "z"])
 (Multivariate polynomial ring in 3 variables over QQ, QQMPolyRingElem[x, y, z])
 
 julia> A, _ = quo(R, ideal(R, [y-x^2, z-x^3]))
-(Quotient of multivariate polynomial ring by ideal (-x^2 + y, -x^3 + z), Map: multivariate polynomial ring -> quotient of multivariate polynomial ring)
+(Quotient of multivariate polynomial ring by ideal (-x^2 + y, -x^3 + z), Map: multivariate polynomial ring -> A)
```

Comment:
The mpoly ring in the map domain has the name `R`. Why is this not used for printing?

Reply:
Can be investigated later.

Reply:
> This may get fixed by [Nemocas/AbstractAlgebra.jl#1589](https://github.com/Nemocas/AbstractAlgebra.jl/pull/1589), but I haven't verified it yet

Yes, indeed